docs/index.md

```diff
@@ -47,48 +49,92 @@ Older plugin that uses less resources than GLideN64, but has visual glitches (do
 
-Used with 3rd-party controllers/adapters, like Xbox/PS3/PS4/Keyboard. There are 3 versions available, and up to preference. You must map your own buttons, and **enable RAW data**.
+=== "Keyboard"
 
+    To use keyboard, first choose *N-Rage's Direct-Input8 1.61* as your input plugin. Then go to Configure Controller Plugin:
+    
+    1. Check Raw Data. **If you skip this step, netplay will not work!**
+    
+    2. Press Assign Keys. In the top right, choose "Keyboard".
 
-### Cheats
+    3. Bind your keys according to preference and choose Apply.
 
-Make sure the `1. Netplay Required` cheat is checked when you set up Project64KSE for the first time. This is a one time process and **does not** need to be done every time you launch the emulator. Start the game first to apply the cheat. If you use cheats other than `1. Netplay Required` you may need to recheck your cheats every time you launch the game.
+=== "Xbox / Other USB"
 
+    If you're using Windows 7, select N-Rage as your controller plugin, then choose Configure Controller Plugin and set your bindings as you like. **Ensure RawData is checked** under controller pak settings 
+    
+    If you're using Windows 8 or newer, you may need to download a premade N-Rage config file. See the [Xbox Controller Configuration](/xbox) page for details.
 
-### Start Netplay
+---
+
+## Start Netplay
+!!! Danger inline end "Ethernet Highly Recommended"
+
+     Use a wired Ethernet connection when playing netplay. WiFi connections are inherently unreliable and will frustrate you and your opponents.
+
+Launch Netplay with `File -> Start Netplay`. Use the `Change Mode` dropdown to select between `Server` or `P2P` (Peer to Peer). Peer to peer connections are more stable, but require one player to port forward in order to host. Higher ping causes higher frame delay. Most players play on 3 frames or less. You **must** restart the emulator if another player is joining or any player changes port. Servers may sometimes go down and active servers may be found on the master server list.
+
+=== "Connecting via Server"
+
+    1. Start Netplay. Server mode is the default.
+    2. Select the server to join. A full list of servers can be found by clicking `Master Servers List`.
+    3. Host or join lobbies to play
+    4. Host starts game
+    
+    Up to 4 players can play in server matches.
+
+=== "Connecting via P2P"
+
+    1. Start Netplay. Click the `Change Mode` dropdown and select `1. P2P`. 
+    2. Host must have ports forwarded and provide external IP to other player
+    3. Host selects the game and clicks `Host`.
+    4. Client chooses the Connect tab, enters the Host's IP in the `Peer IP` box, and connects.
+    5. Both players tick `Ready` to start the game
+    
+    Only 2 players can play P2P.
+
+## Frequently Asked Questions
+
+#### My game runs fine, but the audio is delayed.
```

Comment:
I really don't like the long names when they appear in the Table of Contents. There's a couple ways we could fix it, I don't really care how, other then just having shorter TOC text. We can put them into larger categories, like `Audio`, `Inputs`, `Lag`, `Troubleshooting Help`. Or we could just come up with an abbreviated header that looks nicer on the TOC, using the same trick I used for the plugin headers: `{ #faqaudio data-toc-label='Audio Skipping' }` maybe `{ #faqaudio data-toc-label='Q: Audio Skipping' }`. As long as the TOC header is one line, that's good enough for me.

![image](https://user-images.githubusercontent.com/1178816/211358356-59f76d7a-b7c7-4876-8ab8-e874b2b6e673.png)

Reply:
Fixed using the headers trick in https://github.com/smash64-dev/smash64-online/pull/10/commits/da19be5987fa6d2f4230dea0e60b1096e06ed60f. Had no idea you could do that, works great.